src/endpoints/PricingTables/types.ts

```diff
@@ -0,0 +1,63 @@
+export interface PricingTable {
+    options: Option[];
+}
+
+export enum BillingCycle {
+    MONTH = 'month',
+    YEAR = 'year',
+}
+
+export enum Currency {
+    EUR = 'eur',
+    USD = 'usd',
+}
+
+export enum OptionId {
+    STARTER = 'starter',
+    CORE = 'core',
+    PREMIUM = 'premium',
+}
+
+export enum LimitDisplayName {
+    STORIES = 'Stories',
+    USERS = 'Users',
+    SITES = 'Sites',
+    CONTACTS = 'Contacts',
+    EMAIL_SENDS = 'Email sends',
+    CUSTOM_SENDER_ADDRESSES = 'Custom sender addresses',
+}
+
+export interface Option {
+    id: `${OptionId}`;
```

Comment:
Will this give the SDK problems if we add new pricing option in the API but don't support it in the SDK?

Same question for `Limit.display_name`, `Price.billing_cycle` and `Price.currency`.

Reply:
Then I believe it's best to make it `string`, since this way it will complicate the API updates.

Reply:
Also, most of these enums already exist, see `Billing/types.ts`. I don't think `LimitDisplayName` should be an enum, since backend should have freedom on those.

Reply:
I find it really weird that we prefix those enums in `Billing/types.ts` with `Signup`.

https://github.com/prezly/javascript-sdk/blob/dfd2002b381e9b832b6077669f586ea6a27ea78b/src/endpoints/Billing/types.ts#L1-L16

So in order to access them in frontend we do something like:
```tsx
Billing.SignupPlan
Billing.SignupCurrency
Billing.SignupBillingCycle
```

Makes more sense to just do:
```tsx
Billing.Plan
Billing.Currency
Billing.Cycle
```

But I have removed the PricingTable enums in favor of those: https://github.com/prezly/javascript-sdk/pull/215/commits/dfd2002b381e9b832b6077669f586ea6a27ea78b

Reply:
@MohammadxAli I do not quite understand how adding new enum value can cause TS error. Could you explain a bit more what did you mean?

> Yeah, they will pop up a TS error. Do you think I should change them to string instead? I wanted to be extra type-safe here :)

Reply:
Hey @aspirisen, what I meant was:

```ts
enum SomeEnum {
  OLD_VALUE = 'old'
}

SomeEnum.NEW_VALUE // -> Property 'NEW_VALUE' does not exist on type 'typeof SomeEnum'
```
Because it needs to be added to SDK. I didn't mean _adding new enum value on backend side can cause TS error_ 😬

Reply:
Ah, got what you mean. I think it is fair to update sdk first before using new values in the app, the same we have for objects for example

Reply:
Exactly, I had mixed feelings when deleted enums as well. But I thought we want to give SDK more freedom when we discussed it, but now I realize it was the wrong thing to do.

We always want to update the SDK in the first place.

Reply:
Agree 👍